vite.config.ts

```diff
@@ -4,4 +4,13 @@ import react from "@vitejs/plugin-react";
 // https://vite.dev/config/
 export default defineConfig({
   plugins: [react()],
+  server: {
+    proxy: {
+      '/friendbot': {
+        // Use the local faucet http://localhost:8000/friendbot
+        target: 'http://localhost:8000',
+        changeOrigin: true,
+      },
```

Comment:
Nice use of `proxy`! Can we change the settings based on an environment variable? In `staging` / testnet, the Friendbot URL is `https://friendbot.stellar.org`: https://developers.stellar.org/docs/build/apps/example-application-tutorial/overview#dev-helpers`

Reply:
The local network transitions to the appropriate URL depending on how the `stellar container` is run, so I was thinking of that … but it could be if someone is running in testnet they'd prefer to talk to network friendbot? 🤔